### ISSUE TITLE
Handle editorial elements

### ISSUE THREAD

**Issue by rettinghaus** [trompamusic/mei-friend#4]
Graphical navigation through notes fails within `choice` and `app` elements, because it simply jumps to the next note in the encoding, which is not rendered. Notes that are not rendered by default should be ignored while navigating.

**Comment by wergo** [comment]
Could you provide me a typical MEI example for that?

**Comment by rettinghaus** [comment]
Example with `choice` and `app` elements within a layer. Verovio renders the first child by default.

```xml
<?xml version="1.0" encoding="UTF-8"?>
<?xml-model href="https://music-encoding.org/schema/4.0.1/mei-all.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
<?xml-model href="https://music-encoding.org/schema/4.0.1/mei-all.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
<mei xmlns="http://www.music-encoding.org/ns/mei" meiversion="4.0.1">
   <meiHead>
      <fileDesc>
         <titleStmt>
            <title />
         </titleStmt>
         <pubStmt />
      </fileDesc>
   </meiHead>
   <music>
      <body>
         <mdiv xml:id="mdiv-0000000122406666">
            <score xml:id="score-0000002146985283">
               <scoreDef xml:id="scoredef-0000000781623170">
                  <staffGrp xml:id="staffgrp-0000001093519277">
                     <staffGrp xml:id="staffgrp-0000000613437513">
                        <staffDef xml:id="P1" n="1" lines="5" ppq="1">
                           <clef xml:id="clef-0000000662184591" shape="F" line="4" />
                           <keySig xml:id="keysig-0000001076162183" sig="4s" />
                           <meterSig xml:id="msig-0000000950534647" count="5" unit="4" />
                        </staffDef>
                     </staffGrp>
                  </staffGrp>
               </scoreDef>
               <section xml:id="section-0000000213930840">
                  <measure xml:id="measure-0000000504962096" n="1">
                     <staff xml:id="staff-0000001925267016" n="1">
                        <layer xml:id="layer-0000001826628563" n="1">
                           <note xml:id="note-0000001867524476" dur="4" oct="3" pname="f" accid.ges="s" />
                           <note xml:id="note-0000001881765938" dur="4" oct="3" pname="d" accid.ges="s" />
                           <app xml:id="app-0000000304026057">
                              <lem xml:id="lem-0000000902343786">
                                 <note xml:id="note-0000000617767699" dur="4" oct="2" pname="b" />
                              </lem>
                              <rdg xml:id="rdg-0000001621716379">
                                 <note xml:id="note-0000000324734129" dur="4" oct="2" pname="a" />
                              </rdg>
                           </app>
                           <note xml:id="note-0000001885767495" dur="4" oct="2" pname="a" />
                           <note xml:id="note-0000001530626039" dur="4" oct="3" pname="e">
                              <accid xml:id="accid-0000000525230060" accid="s" accid.ges="s" />
                           </note>
                        </layer>
                     </staff>
                  </measure>
                  <measure xml:id="measure-0000001383829250" right="end" n="2">
                     <staff xml:id="staff-0000001753580705" n="1">
                        <layer xml:id="layer-0000000365337507" n="1">
                           <note xml:id="note-0000000571733376" dur="4" oct="3" pname="f" accid.ges="s" />
                           <choice xml:id="choice-0000001432822013">
                              <orig xml:id="orig-0000001705438680">
                                 <note xml:id="note-0000001474602303" dur="2" oct="3" pname="a" />
                              </orig>
                              <reg xml:id="reg-0000001677187063">
                                 <note xml:id="note-0000000612617319" dur="4" oct="3" pname="a" />
                              </reg>
                           </choice>
                           <choice xml:id="choice-0000001222676715">
                              <reg xml:id="reg-0000000256530862">
                                 <note xml:id="note-0000001514518105" dur="4" oct="3" pname="g" />
                              </reg>
                              <orig xml:id="orig-0000000382122844">
                                 <note xml:id="note-0000001362534578" dur="2" oct="3" pname="g" />
                              </orig>
                           </choice>
                           <note xml:id="note-0000000575209963" dur="4" oct="3" pname="c" accid.ges="s" />
                           <note xml:id="note-0000000538873425" dur="4" oct="2" pname="f" accid.ges="s" />
                        </layer>
                     </staff>
                  </measure>
               </section>
            </score>
         </mdiv>
      </body>
   </music>
</mei>
```

**Comment by wergo** [comment]
This is implemented. As soon as Verovio 3.7 is released (and the npm package available), a new version of mei-friend will be published.

**Comment by wergo** [comment]
Fixed in mei-friend 0.6.0.
